package.json

```diff
@@ -45,43 +45,43 @@
       }
     ],
     "okapiInterfaces": {
-      "inventory": "10.10 11.0 12.0",
-      "search": "0.7",
-      "source-storage-records": "3.0",
-      "instance-storage": "7.0 8.0 9.0",
+      "alternative-title-types": "1.0",
+      "call-number-types": "1.0",
+      "circulation": "9.0 10.0 11.0 12.0 13.0",
+      "classification-types": "1.1",
+      "configuration": "2.0",
+      "contributor-name-types": "1.2",
+      "contributor-types": "2.0",
+      "data-export": "5.0",
+      "electronic-access-relationships": "1.0",
+      "holdings-note-types": "1.0",
+      "holdings-sources": "1.0",
       "holdings-storage": "3.0 4.4 5.0 6.0",
+      "holdings-types": "1.0",
+      "identifier-types": "1.1",
+      "ill-policies": "1.0",
+      "instance-formats": "2.0",
+      "instance-note-types": "1.0",
+      "instance-relationship-types": "1.0",
+      "instance-statuses": "1.0",
+      "instance-storage": "7.0 8.0 9.0",
+      "instance-types": "2.0",
+      "inventory-record-bulk": "1.0",
+      "inventory": "10.10 11.0 12.0 13.0",
+      "item-note-types": "1.0",
       "item-storage": "8.0 9.0 10.0",
       "loan-types": "2.0",
-      "material-types": "2.0",
-      "item-note-types": "1.0",
+      "location-units": "2.0",
       "locations": "3.0",
-      "identifier-types": "1.1",
-      "contributor-types": "2.0",
-      "contributor-name-types": "1.2",
-      "instance-types": "2.0",
+      "material-types": "2.0",
+      "modes-of-issuance": "1.0",
       "nature-of-content-terms": "1.0",
-      "instance-formats": "2.0",
-      "classification-types": "1.1",
+      "search": "0.7",
```

Comment:
Looks like you forgot to update the interface to support version `1.0`

Reply:
@usavkov-epam good catch! Thank you.